### DISCUSSION TITLE
fixed gap size between grid elements across all screensizes?

### DISCUSSION THREAD

**Discussion by gqoew** [tailwindlabs/tailwindcss#5849] [Help]
I created a grid as follow:

```html
<div class="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 lg:grid-cols-5 gap-x-8 gap-y-8">
  <Item
    v-for="object in objects"
    :key="object.id"
    :object="object"
  />
</div>
```

I am struggling to keep the `gap-x-8` fixed between each item. It will grow as soon as it has space available. How can I keep the gap identical between grid elements whatever the every screen size?

**Comment by leevigraham** [comment]
Gap will be the same (checkout browser debugging). What I think is happening is that your <Item> has another div in it which is changing the visual gap.
